contrib/perftest/main.go

```diff
@@ -0,0 +1,261 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/containers/image/types"
+	"github.com/containers/libpod/libpod"
+	image2 "github.com/containers/libpod/libpod/image"
+	cc "github.com/containers/libpod/pkg/spec"
+	"github.com/containers/libpod/pkg/util"
+	"github.com/containers/storage/pkg/reexec"
+	"github.com/cri-o/ocicni/pkg/ocicni"
+	"github.com/pkg/profile"
+	"github.com/sirupsen/logrus"
+)
+
+/*
+const (
+	testImageName = "docker.io/library/alpine:latest"
+	testRunCount  = 50
+)*/
+
+var helpMessage = `
+-count int
+	count of loop counter for test (default 50)
+-image string
+	image-name to be used for test (default "docker.io/library/alpine:latest")
+`
+
+func main() {
+
+	ctx := context.Background()
+	imageName := ""
+
+	testImageName := flag.String("image", "docker.io/library/alpine:latest", "image-name to be used for test")
+	testRunCount := flag.Int("count", 50, "count of loop counter for test")
+	flag.Usage = func() {
+		fmt.Fprintf(os.Stderr, "Usage of %s:\n", os.Args[0])
+		fmt.Fprintf(os.Stderr, "%s \n", helpMessage)
+	}
+	flag.Parse()
+
+	if reexec.Init() {
+		return
+	}
+
+	logrus.SetLevel(logrus.ErrorLevel)
```

Comment:
A `--log-level` flag might come in handy for debugging the tool.

Reply:
done :)